package.json

```diff
@@ -122,6 +122,9 @@
     "typescript": "^5",
     "ws": "^8.17.1"
   },
+  "resolutions": {
+    "strip-ansi": "6.0.1"
+  },
```

Comment:
We were running into: https://github.com/eslint/eslint/discussions/17215#discussioncomment-7884236